lib/vanagon/platform/defaults/sles-11-x86_64.rb

```diff
@@ -0,0 +1,11 @@
+platform "sles-11-x86_64" do |plat|
+  plat.servicedir "/usr/lib/systemd/system"
+  plat.defaultdir "/etc/sysconfig"
+  plat.servicetype "systemd"
+
+  plat.add_build_repository "http://pl-build-tools.delivery.puppetlabs.net/yum/pl-build-tools-release-#{plat.get_os_name}-#{plat.get_os_version}.noarch.rpm"
+  packages = %w(aaa_base autoconf automake rsync gcc make rpm-build)
+  plat.provision_with "zypper -n --no-gpg-checks install -y #{packages.join(' ')}"
+  plat.install_build_dependencies_with "zypper -n --no-gpg-checks install -y"
+  plat.vmpooler_template "sles-11-x86_64"
+end
```

Comment:
I don't feel that we should do this. Any sles-11 configuration should be added to the individual projects rather than generically supported by vanagon.

Reply:
Got it @e-gris ! I have resolved other suggestions.